RestApi/Sepes.RestApi/Model/Study.cs

```diff
@@ -44,5 +47,25 @@ public class Study
                 archived = archived
             };
         }
+        
+        public override bool Equals(object obj)
+        {
+            return obj is Study study &&
+                   studyName == study.studyName &&
+                   studyId == study.studyId &&
+                   Enumerable.SequenceEqual(pods, study.pods) &&
+                   Enumerable.SequenceEqual(sponsors, study.sponsors) &&
+                   Enumerable.SequenceEqual(suppliers, study.suppliers) &&
+                   Enumerable.SequenceEqual(datasets, study.datasets) &&
+                   archived == study.archived &&
+                   Enumerable.SequenceEqual(userIds, study.userIds) &&
+                   Enumerable.SequenceEqual(datasetIds, study.datasetIds);
+        }
+
+        public override int GetHashCode()
+        {
+            return HashCode.Combine(studyName, studyId, pods, sponsors, suppliers, archived, userIds, datasetIds);
```

Comment:
Cant use pods, sponsors, suppliers, userIds or datasetIds as part of hash. See previous issue.